docs/index.rst

```diff
@@ -4,47 +4,64 @@
    contain the root `toctree` directive.
 
 Welcome to LlamaIndex 🦙 !
-=====================================
+##########################
 
-LlamaIndex (GPT Index) is a data framework for your LLM application.
```

Comment:
Sadly, we get quite a few people still searching the old name on google. Should probably keep this for now to help with searching tbh

Reply:
makes sense, added back